rules/falco_rules.yaml

```diff
@@ -3398,6 +3398,38 @@
   priority: WARNING
   tags: [host, container, mitre_execution, mitre_defense_evasion, T1036.005, T1059.004]
 
+#The given rule is designed to detect suspicious activities related to disabling system recovery features. 
+#The rule looks for specific commands being executed such as rm -rf, chattr +i, mkfs.ext4 -E nodiscard, mount remount,ro, 
+#and disabling certain services such as systemd-backlight@.service, apport.service, rescue.service, emergency.service, and recovery-mode.service. 
+#https://attack.mitre.org/techniques/T1490/
+#Mitre Impact: Inhibit system recovery
+
+- list: recovery_files
+  items: [/etc/default/grub, /etc/inittab, /usr/lib/systemd/system/rescue.service]
+  
+- list: critical_dir
+  items: [/boot, /bin, /etc, /lib, /sbin, /usr, ~/.local/share/Trash, ~/.local/share/Trash/files]
+
+- rule: Disable Recovery Features
+  desc: Detects disabling system recovery features by deleting or disabling services and commands.
+  condition: >
+      (spawned_process) and 
+      ((open_write and fd.name in (recovery_files)) or
+      (remove and (fd.name in (critical_dir))) or
+      (proc.name = "chattr" and proc.args contains "+i") or
+      (proc.name = "mkfs.ext4" and proc.args contains "-E nodiscard") or
+      (proc.name = "mount" and proc.args contains "remount,ro") or
+      (proc.name = "systemctl" and proc.args contains "disable systemd-backlight@.service") or 
+      (proc.name = "systemctl" and proc.args contains "disable apport.service") or
+      (proc.name = "systemctl" and proc.args contains "disable rescue.service") or
+      (proc.name = "systemctl" and proc.args contains "disable emergency.service") or
+      (proc.name = "systemctl" and proc.args contains "disable recovery-mode.service"))
```

Comment:
That would become:

### Condition rule 1
```
spawned_process and 
((proc.name = chattr and proc.args contains +i) or
(proc.name = "mkfs.ext4" and proc.args contains "-E nodiscard") or
<redacted, everything with proc.name here>)
```

### Condition rule 2
```
open_write and fd.name in (recovery_files)
```

### Condition rule 3 
```
remove and (fd.name in (critical_dir))
```
here we could also think to delete the macro and use operator `pmatch`, to match on the directory prefixes, and trigger if also files are deleted and not the directory as a whole

Reply:
@loresuso yep. we can do that. let me send these changes as well.

Reply:
so can we split the rule in three different ones? I prefer to have them separate once a rule takes into account many scenarios like this one.